activiti-cloud-services-org/activiti-cloud-services-org-core/pom.xml

```diff
@@ -22,5 +21,10 @@
       <groupId>org.springframework.data</groupId>
       <artifactId>spring-data-rest-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.activiti</groupId>
+      <artifactId>activiti-process-validation</artifactId>
+      <version>${version}</version>
```

Comment:
version should be set by parent and removed here

Reply:
done